### PR TITLE
#1 - daonguyendnv created gallerySlice.js file

### PR DESCRIPTION
**Pull request by daonguyendnv**
I do! refer #1

### CODE REVIEW

**Review by badaovz — COMMENTED** [review]
i saw

File: src/gallerySlice.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    return images;
+})
+
+const gallerySlice = createSlice({
+    name: 'gallery',
+    initialState: {
+        images: [],
+        isLoading: flase
+    },
+    extraReducers: (builder) => {
```
Comment (badaovz) [inline src/gallerySlice.js:16]: I saw this line

**Review by badaovz — COMMENTED** [review]
review #1

File: src/gallerySlice.js (hunk 1 above)
Comment (daonguyendnv) [reply src/gallerySlice.js:16]: yes

**Comment by badaovz** [comment]
good!

**Pull request merged**